installation/resources/kyma/installer-cr-kyma.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: "installer.kyma-project.io/v1alpha1"
```

Comment:
This file and `installation/resources/kyma/installer-overrides-kyma-config-local.yaml` were previously downloaded each time. Now we'll use local copies - is there reason to be concerned that we might miss out on some kyma configuration changes this way or do we belief this configuration should fit our needs for a long time

Reply:
Im not sure, I was thinking to keep them locally for now as this one is slightly reduced (I removed some components such as kiali, tracing, etc).  Also, we currently have the minimal kyma CR locally, so for consistency reasons i thought we could use the same approach here for the full kyma CR.

Maybe if we see that next kyma releases change the format, etc we should think about downloading them each time (both minimal and full CRs) and also downloading the kyma cli that is used with the correct version (matching the version specified in kyma_version file or as --kyma-release arg). This would probably require doing a docker-in-docker installation so that we don't interfere with any kyma cli already installed on the host.